optimum/onnxruntime/io_binding/io_binding_helper.py

```diff
@@ -140,7 +140,7 @@ def to_pytorch_via_cupy(ort_value: OrtValue) -> torch.Tensor:
     def to_pytorch_via_dlpack(ort_value: OrtValue) -> torch.Tensor:
         from torch._C import _from_dlpack
 
-        torch_tensor = ort_value.to_dlpacks(_from_dlpack)
+        torch_tensor = _from_dlpack(ort_value.to_dlpack())
```

Comment:
What is the difference?

Reply:
No difference technically, just a change to adapt the current API in onnxruntime.